srcs/Server.cpp

```diff
@@ -66,7 +66,8 @@ void Server::executor()
 		//std::cout << "Executing: " << current->getCommand()  << " TXT: " << current->getParams()[0] << std::endl; 
 		// ^^ this causes a segfault if there are no params
 		if (current->getComCategory() != IGNORE && current->getType() != UNKNOWN)
-			test[current->getComCategory()][current->getType()](*this, *current);
+			if (test[current->getComCategory()][current->getType() % 10] != NULL)
+				test[current->getComCategory()][current->getType() % 10](*this, *current);
```

Comment:
In what case is the modulo necessary? (assuming the parser assigns commands appropriately)

Reply:
Forgot about the way we divided the commands in Commands.hpp, good catch